Doesn't compile against Arduino-ESP32 3.3.2

The example

https://github.com/earlephilhower/ESP8266Audio/tree/2.3.0/examples/PlayWAVFromPROGMEM

Fails to compile with 

```
"C:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp-x32\\2507/bin/xtensa-esp32-elf-gcc" -MMD -c "@C:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.5-07e9bf49-v1\\esp32/flags/c_flags" -w -Os -Werror=return-type -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_ESP32_DEV -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"ESP32_DEV\"" "-DARDUINO_VARIANT=\"esp32\"" -DARDUINO_PARTITION_default "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:esp32:UploadSpeed=921600,CPUFreq=240,FlashFreq=80,FlashMode=qio,FlashSize=4M,PartitionScheme=default,DebugLevel=none,PSRAM=disabled,LoopCore=1,EventsCore=1,EraseFlash=none,JTAGAdapter=default,ZigbeeMode=default\"" -DESP32=ESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DARDUINO_USB_CDC_ON_BOOT=0 "@C:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.5-07e9bf49-v1\\esp32/flags/defines" "-IC:\\Users\\Max\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2025918-12796-3c7u1e.h9sw4\\PlayWAVFromPROGMEM" -iprefix "C:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.5-07e9bf49-v1\\esp32/include/" "@C:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.5-07e9bf49-v1\\esp32/flags/includes" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.5-07e9bf49-v1\\esp32/qio_qspi/include" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\cores\\esp32" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\variants\\esp32" "-Ic:\\Users\\Max\\Documents\\Arduino\\libraries\\ESP8266Audio\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\FS\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\SPIFFS\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\HTTPClient\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\Network\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\NetworkClientSecure\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\SD\\src" "-IC:\\Users\\Max\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.3.2\\libraries\\SPI\\src" "@C:\\Users\\Max\\AppData\\Local\\arduino\\sketches\\4CA741211C4721F47487206B5BB5F865/build_opt.h" "@C:\\Users\\Max\\AppData\\Local\\arduino\\sketches\\4CA741211C4721F47487206B5BB5F865/file_opts" "c:\\Users\\Max\\Documents\\Arduino\\libraries\\ESP8266Audio\\src\\libflac\\window.c" -o "C:\\Users\\Max\\AppData\\Local\\arduino\\sketches\\4CA741211C4721F47487206B5BB5F865\\libraries\\ESP8266Audio\\libflac\\window.c.o"
c:\Users\Max\Documents\Arduino\libraries\ESP8266Audio\src\AudioOutputPDM.cpp: In member function 'virtual bool AudioOutputPDM::begin()':
c:\Users\Max\Documents\Arduino\libraries\ESP8266Audio\src\AudioOutputPDM.cpp:74:5: error: 'i2s_pdm_tx_gpio_config_t' has no non-static data member named 'dout2'
   74 |     };
      |     ^

```

In the Arduino IDE with settings

<img width="591" height="662" alt="Image" src="https://github.com/user-attachments/assets/13ff0fdb-149a-467a-b205-f9dadabbdf1a" />

--- a/src/AudioOutputPDM.cpp
+++ b/src/AudioOutputPDM.cpp
@@ -66,7 +66,9 @@ bool AudioOutputPDM::begin() {
         .gpio_cfg = {
             .clk = I2S_GPIO_UNUSED,
             .dout = (gpio_num_t)pdmPin,
+#if SOC_I2S_PDM_MAX_TX_LINES > 1
             .dout2 = I2S_GPIO_UNUSED,
+#endif
             .invert_flags = {
                 .clk_inv = false,
             },

